doc/style-guide.md

```diff
@@ -525,6 +525,62 @@ word within a compound word is capitalized except for the first word.
 * Any exceptions to these conventions should be flagged on the GitHub
   `agda-stdlib` issue tracker in the usual way.
 
+#### Fixity
+
+All functions and operators that are not purely prefix (typically
+anything that has a `_` in its name) should have an explicit fixity
+declared for it. The guidelines for these are as follows:
+
+General operations and relations:
+
+* binary relations of all kinds are `infix 4`
+
+* unary prefix relations `infix 4 ε∣_`
+
+* unary postfix relations `infixr 8 _∣0`
+
+* multiplication-like: `infixl 7 _*_`
+
+* addition-like  `infixl 6 _+_`
+
+* negation-like  `infix  8 ¬_`
```

Comment:
Aaaargh!
* `Relation.Nullary.Negation.Core` declares `infix 3 ¬_`, and is (definitely!?) a special case
* `Data.Integer.Base.-_` etc. otherwise follow the pattern
```suggestion
* unary prefix negation-like  `infix  8 -_` (NB. *logical* negation `infix 3 ¬_` is a special case)
```

Reply:
And ... what about an additional/separate entry for *binary* `_-_` (as `infixl 6 `_-_`) as in `Algebra.Structures.IsGroup` and `Data.Integer.Base`?

Reply:
Good points. I'm using slight variants of the ideas, but basically 'yes' to both.